main.html

```diff
@@ -134,122 +177,204 @@ <h1 class="let-ride__header">Let's ride</h1>
+                                    <img src="man.png" class="ride__earth" alt="ride__earth"/>
+                                    <p>Private</p>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                    <div class="ride__card ride__card--lyft">
+                        <section class="ride__rentals">
+                            <span class="ride__card-header">Lyft </span>
+                            <img src="Lyft__car.png" class="ride__car" alt="Ride__Rentals"/>
+                            <div class="ride__card-points">
+                                <div class="ride__card-point">
+                                    <img src="thunder.png" class="ride__keys" alt="ride__keys"/>
+                                    <p>Efficient</p>
+                                </div>
+                                <div class="ride__card-point">
+                                    <img src="man.png" class="ride__earth" alt="ride__earth"/>
+                                    <p>Private</p>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                </div>
+            </div>
+            <p class="ride-cards__text">
+                *Availability of bikes, scooters, rentals, ride types, and transit
                 information varies by region.
                 </p>
-            </div>
         </div>
         <div class="estimation">
             <div class="estimation__container">
-                <p class="estimation__header"> Ready to roll? <br> Get an estimate </p>
+                <p class="estimation__header">Ready to roll? <br> Get an estimate. </p>
                     <div class="estimation__textareas">
-                        <textarea rows="3" cols="1" placeholder="Enter pick-up location*" class="estimation_textarea"></textarea>
+                        <textarea rows="1" cols="1" placeholder="Enter pick-up location*" class="estimation__textarea"></textarea>
                         <div class="estimation__points">
-                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="estimation__point" viewBox="0 0 16 16">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="estimation__point estimation__point--black" viewBox="0 0 16 16">
                                 <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-8 3a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"/>
                             </svg>
                         </div>
-                        <textarea rows="3" cols="1" placeholder="Enter drop-off location*" class="estimation_textarea"></textarea>
+                        <textarea rows="1" cols="1" placeholder="Enter drop-off location*" class="estimation__textarea"></textarea>
```

Comment:
use input instead, textarea is for multirow stuff

Reply:
+